client/src/featureform/cli.py

```diff
@@ -125,17 +122,14 @@ def list(host, cert, insecure, local, resource_type):
         if host == None:
             raise ValueError(
                 "Host value must be set with --host flag or in env as FEATUREFORM_HOST")
-                
-    if insecure:
-        rc = ResourceClient(host, False)
-    else:
-        rc = ResourceClient(host, True, cert)
+    
+    rc = ResourceClient(host=host, local=local, insecure=insecure, cert_path=cert)
 
     rc_list_functions = {
         "features": rc.list_features,
         "labels": rc.list_labels,
         "sources": rc.list_sources,
-        "trainingsets": rc.list_sources,
+        "trainingsets": rc.list_training_sets,
```

Comment:
why is there trainingsets and training-sets?

Reply:
just in case they have a typo, but should i remove one?